.github/workflows/pipeline-docker-cloud.yaml

```
@@ -119,7 +126,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ env.DOCKERHUB_ORG_NAME }}/${{ env.IMAGE_NAME }}
+          images: ${{ env.DOCKERHUB_ORG_NAME }}/${{ env.REPOSITORY_OWNER }}-${{ env.IMAGE_NAME_SUFFIX }}
```

```suggestion
          images: ${{ env.DOCKERHUB_NAMESPACE }}/${{ env.REPOSITORY_OWNER }}-${{ env.IMAGE_NAME_SUFFIX }}
```

Using namespace in case a personal namespace is being used.